diff --git a/src/board-gcw0.c b/src/board-gcw0.c
index 5fce5ce..45eee3a 100644
--- a/src/board-gcw0.c
+++ b/src/board-gcw0.c
@@ -25,7 +25,7 @@
 
 
 #define FVCO_MIN  300*1000*1000	/* 300 MHz */
-#define FVCO_MAX  1000*1000*1000	/* 1 GHz */
+#define FVCO_MAX  1000*1000*1200	/* 1.2 GHz */
 
 /* The frequency after the input divider must be between 10 and 50 MHz.
 The highest divider yields the best resolution. */
